Cargo.lock

```diff
@@ -1,7185 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
We shouldn't delete the lock file

Reply:
Looks like rebase is still needed too.

Reply:
Working on it - it has happened after my merge...

Reply:
Done